zhaquirks/tuya/air/ts0601_air_quality.py

```diff
@@ -103,7 +102,7 @@ class TuyaCO2SensorGPP(CustomDevice):
                 PROFILE_ID: 41440,
                 DEVICE_TYPE: 97,
                 INPUT_CLUSTERS: [],
-                OUTPUT_CLUSTERS: [GreenPowerProxy.cluster_id],
+                OUTPUT_CLUSTERS: [33],
```

Comment:
Why?
I think is better keep the `GreenPowerProxy` reference

Reply:
+1